tests/test_sensors.py

```diff
@@ -0,0 +1,175 @@
+
+    
+    # def test_round_1D(self):
+    #     def f(x): return np.zeros(x.shape)
+    #     sensor = RoundSensor(0, f, 0.4, np.array([[-5], [5]]), 1, 1)
+        
+    #     relative_pos = sensor.get_sites()
+    #     actual_pos = np.array([5])
+    #     measure_pos = relative_pos + actual_pos*np.ones(relative_pos.shape)
+    #     true_pos = np.array([[5], [5], [5], [4], [6]])
+    #     for i, pos in enumerate(measure_pos):
+    #         self.assertEqual(pos[0], true_pos[i, 0])
+
+    #     true_values = np.array([[1], [2], [3], [4], [5]])
+    #     sensor.set_values(true_values)
+    #     sensor_value = sensor.get_values()
+    #     self.assertEqual(sensor_value[0, 0], 3)
+    
+
+    # def test_round_2D(self):
+    #     def f(x): return np.zeros(x.shape)
+    #     sensor = RoundSensor(0, f, 0.4, np.array([[-5], [5]]), 1, 2)
+        
+    #     relative_pos = sensor.get_sites()
+    #     actual_pos = np.array([5, 5])
+    #     measure_pos = relative_pos + actual_pos*np.ones(relative_pos.shape)
+    #     true_pos = np.array([[5, 5], [5, 6], [5, 4], [4, 5], [6, 5]])
+    #     for i, pos in enumerate(measure_pos):
+    #         self.assertEqual(pos[0], true_pos[i, 0])
+    #         self.assertEqual(pos[1], true_pos[i, 1])
+
+    #     true_values = np.array([[1]])
+    #     sensor.set_values(true_values)
+    #     sensor_value = sensor.get_values()
+    #     self.assertEqual(sensor_value[0, 0], 1)
+
+
+    # def test_thermo_1D(self):
+    #     temps = np.linspace(-10, 10, 20).reshape(-1, 1)
+    #     voltages = temps.copy()
+    #     sensor = Thermocouple(temps, voltages, 1)
+
+    #     relative_pos = sensor.get_sites()
+    #     actual_pos = np.array([5])
+    #     measure_pos = relative_pos + actual_pos*np.ones(relative_pos.shape)
+    #     true_pos = np.array([[5], [5], [5], [4.99925], [5.00075]])
+    #     for i, pos in enumerate(measure_pos):
+    #         self.assertEqual(pos[0], true_pos[i, 0])
+
+    #     true_values = np.array([[1], [2], [3], [4], [5]])
+    #     sensor.set_values(true_values)
+    #     sensor_value = sensor.get_values()
+    #     self.assertAlmostEqual(sensor_value[0, 0], 3, -3)
+
+    
+    
+    # def test_thermo_2D(self):
+    #     temps = np.linspace(-10, 10, 20).reshape(-1, 1)
+    #     voltages = temps.copy()
+    #     sensor = Thermocouple(temps, voltages, 2)
+
+    #     relative_pos = sensor.get_sites()
+    #     actual_pos = np.array([5, 5])
+    #     measure_pos = relative_pos + actual_pos*np.ones(relative_pos.shape)
+    #     true_pos = np.array([[5, 5], [5, 5.00075], [5, 4.99925], [4.99925, 5], [5.00075, 5]])
+    #     for i, pos in enumerate(measure_pos):
+    #         self.assertEqual(pos[0], true_pos[i, 0])
+
+    #     true_values = np.array([[1], [2], [3], [4], [5]])
+    #     sensor.set_values(true_values)
+    #     sensor_value = sensor.get_values()
+    #     self.assertAlmostEqual(sensor_value[0, 0], 3, -3)
```

Comment:
If these tests are no longer in use, please delete them. If you wish to skip them, you may use a skip decorator along with a string providing the reason for the skip: https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures